updating/understanding-upgrade-channels-release.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} 4.1, Red Hat introduced the concept of channels for recommending the appropriate release versions for cluster updates. By controlling the pace of updates, these update channels allow you to choose an update strategy. Upgrade channels are tied to a minor version of {product-title}. For instance, {product-title} 4.10 upgrade channels recommend update to 4.10 and updates within 4.10. They also recommend updates within 4.9 and from 4.9 to 4.10, to allow clusters on 4.9 to eventually update to 4.10. They do not recommend updates to 4.11 or later releases. This strategy ensures that administrators explicitly decide to update to the next minor version of {product-title}.
+In {product-title} 4.1, Red Hat introduced the concept of channels for recommending the appropriate release versions for cluster updates. By controlling the pace of updates, these upgrade channels allow you to choose an update strategy. Upgrade channels are tied to a minor version of {product-title}. For instance, {product-title} 4.9 upgrade channels recommend updates to 4.9 and updates within 4.9. They also recommend updates within 4.8 and from 4.8 to 4.9, to allow clusters on 4.8 to eventually update to 4.9. They do not recommend updates to 4.10 or later releases. This strategy ensures that administrators explicitly decide to update to the next minor version of {product-title}.
```

Comment:
You'll need to revert these version changes (it should stay 4.9 and 4.10).

I'll do a PR to main tomorrow to update main to use 4.9 and 4.10